src/lib/modules/scaffolding/framework/react/index.ts

```diff
@@ -0,0 +1,101 @@
+import Handlebars from "handlebars";
+import * as path from "path";
+
+import { Contract } from "../../../../../typings/contract";
+import { Embark } from "../../../../../typings/embark";
+import { Builder } from "../../builder";
+import { Description } from "../../description";
+import { Options } from "../../options";
+
+const fs = require("../../../../core/fs");
+const utils = require("../../../../utils/utils");
+require("../../handlebarHelpers");
+
+const indexTemplatePath = path.join(__dirname, "templates", "index.html.hbs");
+const dappTemplatePath = path.join(__dirname, "templates", "dapp.js.hbs");
```

Comment:
Just a tiny one, but maybe we want to put those arbitrary file names into constants as well. E.g. `HANDLEBARS_INDEX_TEMPLATE` and `HANDLEBARS_DAPP_TEMPLATE` or sth. like this..

Reply:
They are not used anywhere else and they are constants, what is the advantages?

Reply:
Yea, fair enough. I personally prefer keeping concrete strings out of routines. But not a biggie :)